### ISSUE TITLE
Not able to use tsconfig paths in tests

### ISSUE THREAD

**Issue by shashikiran797** [BugDiver/gauge-ts#38]
**Describe the bug**

**To Reproduce**
1. Create a starter project for gauge with vscode extension
2. Go to tsconfig.json
3. Add baseUrl and paths parameter
```{
  "compilerOptions": {
    "baseUrl": ".", // This must be specified if "paths" is.
    "paths": {
      "jquery": ["node_modules/jquery/dist/jquery"] // This mapping is relative to "baseUrl"
    }
  }
}
```
4. Try to import the name declared in paths like 
```
import * as jq from 'jquery'
```

Throws error:

```
{ Error: Cannot find module 'jquery'
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
at Function.Module._load (internal/modules/cjs/loader.js:508:25)
at Module.require (internal/modules/cjs/loader.js:637:17)
at require (internal/modules/cjs/helpers.js:22:18)
```

**Expected behavior**
Should not throw any error

**Desktop (please complete the following information):**
 - OS: Mac
 - Gauge and plugin version :
```
Gauge version: 1.0.6

Plugins
-------
html-report (4.0.8)
screenshot (0.0.1)
ts (0.0.2)
```
 - Node version: v10.15.3
 - Npm version : **(I am using yarn instead of npm)**
yarn version: 1.21.1

**Additional context**
For example if you are facing an issue with editor (vscode)
 - VS Code Version : 1.41.1

**Comment by BugDiver** [comment]
@shashikiran797 Merged your PR. But when you run a project a project and the `tsconfig-paths` is not installed it creates an error. tough this case is handled but it creates some unnecessary logs (exception). It can mislead users.

To avoid this I was thinking of doing one of the following
* Remove the error logging
* Add `config-paths` as a required dependencies

What would you prefer?

**Comment by shashikiran797** [comment]
@pocketcowboy 

FYI for @BugDiver 's comment

**Comment by BugDiver** [comment]
The Fix is available in v0.0.4
